frontend/src/pages/team/Library/routes.js

```diff
@@ -2,6 +2,6 @@ import Blueprints from './Blueprints.vue'
 import TeamLibrary from './TeamLibrary.vue'
 
 export default [
+    { name: 'LibraryTeamLibrary', path: 'team-library/:entryPath*', component: TeamLibrary },
```

Comment:
I think this change can be reverted as it has no impact after changing the default redirect

Reply:
Yes, I think so. Let me revert this change